### ISSUE TITLE
Immich implementation

### ISSUE THREAD

**Issue by vednolacni** [spocky/miproja1#180]
I would love to see an option that allows Immich to integrate with wallpaper service to automatically shuffle photos from a specified album.

Immich: https://github.com/immich-app/immich

There is an app for android TV already: https://github.com/giejay/Immich-Android-TV

Thank you for considering this feature request, and please let me know if there’s any other information I can provide to support its implementation!

Best regards.

**Comment by manuel-rw** [comment]
I'd be interested in this too 👍 
Perhaps as a workaround we can download the album and use the "wallpaper from folder".

**Comment by CalmSeoul** [comment]
Yeah it would be nice to use an immich album as a centralized place to manage the wallpapers for multiple tvs all running projectivity
